.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @leonm1
```

Comment:
Please add mpwarres@ and martijneken@ (on one line) so we have multiple reviewers.

Reply:
Good point about the Rust SDK, but two wrongs don't make a right :-)

I think a separate CODEOWNERS mechanism makes sense:
- This is more fine grained than admin/write access
- This is more public / discoverable than admin/write access
- Admin access need not imply code/review expertise

For comparison, the Envoy policy is to have 2+ owners per component: https://github.com/envoyproxy/envoy/blob/main/CODEOWNERS. I think that's a decent rule. I'm qualified to be your secondary here given I do have Go expertise ("readability") even if I haven't worked in this repo.

Reply:
Discussed offline.